pages/desktop/frontend/versions.py

```diff
@@ -23,3 +30,94 @@ def versions_page_header(self):
     def latest_version_number(self):
         el = self.find_element(*self._latest_version_locator).text
         return el.split()[1].replace('Version ', '')
+
+    @property
+    def notice_message(self):
+        return self.find_elements(*self._notice_message_locator)[0]
+
+    @property
+    def notice_message_text(self):
+        text = self.find_elements(*self._notice_message_text_locator)[0].text
+        text += self.find_elements(*self._notice_message_text_locator)[1].text
+        return text
+
+    @property
+    def rating_card(self):
+        el = self.find_element(By.CLASS_NAME, 'AddonSummaryCard')
+        return RatingStats(self, el)
+
+    @property
+    def latest_version(self):
+        return self.VersionCard(
+            self, self.find_elements(*self._versions_list_locator)[0]
+        )
+
+    @property
+    def versions_list(self):
+        items = self.find_elements(*self._versions_list_locator)
+        return [self.VersionCard(self, el) for el in items]
+
+    class VersionCard(Region):
+        _version_number_locator = (By.CSS_SELECTOR, '.AddonVersionCard-version')
+        _released_date_locator = (By.CSS_SELECTOR, '.AddonVersionCard-fileInfo')
+        _version_release_notes_locator = (
+            By.CSS_SELECTOR,
+            '.AddonVersionCard-releaseNotes',
+        )
+        _license_link_locator = (By.CSS_SELECTOR, '.AddonVersionCard-license > a')
+        _license_text_locator = (By.CSS_SELECTOR, '.AddonVersionCard-license')
+        _warning_message_locator = (By.CSS_SELECTOR, '.Notice-text')
+        _warning_learn_more_button_locator = (By.CSS_SELECTOR, '.Notice-button')
+        _add_to_firefox_button_locator = (By.CSS_SELECTOR, '.AMInstallButton-button')
+        _download_link_locator = (
+            By.CSS_SELECTOR,
+            '.InstallButtonWrapper-download-link',
+        )
+
+        @property
+        def version_number(self):
+            return self.find_element(*self._version_number_locator).text[8:]
```

Comment:
Use a text split here instead of a list slice.

Reply:
Ok.